src/dependencies.jl

```diff
@@ -864,15 +864,15 @@ macro install(_libmaps...)
                         depsfile_buffer = IOBuffer()
                         println(depsfile_buffer,
                             """
-                            # This is an auto-generated file; do not edit
+                            # This is an auto-generated file; do not edit this file.
```

Comment:
Oh I see, that makes sense. I think it is still worthy to update and remove the dead code here, even if it does force rebuilding the code. I think this is reasonable since it will be triggered upon a an update in the package. And hopefully this will not need to be changed again in the future.